Extension/src/SSH/commandInteractors.ts

```diff
@@ -4,7 +4,14 @@
  * ------------------------------------------------------------------------------------------ */
 
 import * as vscode from 'vscode';
-import { stripEscapeSequences, isWindows, escapeStringForRegex } from '../common';
+import { stripEscapeSequences, isWindows, escapeStringForRegex, ISshHostInfo, getFullHostAddress, extensionContext } from '../common';
+
+/**
+ * Users' passwords already used for auth.
+ * If a user's pass is already used and yet we still get the same prompt, we probably got a wrong password.
+ * Needs to be reset for each command.
+ */
+export const userUsedPasswords: Set<string> = new Set<string>();
```

Comment:
nit: Naming suggests this a set of passwords. But it is a collection of users who have inputted passwords before.

I would name it `knownPasswordUsers` and clean-up the description to be:
```
The users that have already inputted a password for auth.
```

Reply:
Ah I see. I'll name it "inputtedPasswordUsers". Since this isn't for users that has their passwords cached, but for users we inputted passwords automatically.